### DISCUSSION TITLE
Minting DAPP text-error-txt + can't find stylesheet import

### DISCUSSION THREAD

**Discussion by ClydeMaster** [hashlips-lab/nft-erc721-collection#462] [Q&A]
Hello everyone,
Can you please help me?
Followed the masterclass and everything went fine. Minting Dapp is live, tested, worked fine. (THANKS again Liarco and Daniel for this great job!).
Now I changed the text color of the error warning in the DAPP ("...Could not detect metamask ......") in the tailwind.config as I've done before.
Wanted to check again with yarn dev-server and got the error below. I have not even touched any other files!l
I have customized the DAPP wit h my colors before and this error has never occured.
I urgently need to fix this as I have a Whitelist mint soon and need to update the whitelist jason and run yarn build for the DAPP again.
Really would appreciate your help!

<img width="1372" alt="grafik" src="https://github.com/hashlips-lab/nft-erc721-collection/assets/104903064/c3119c78-bb66-4069-9887-8ef6b02182f7">
<img width="770" alt="grafik" src="https://github.com/hashlips-lab/nft-erc721-collection/assets/104903064/a09bd749-3423-44fc-9191-5341c75c8620">
<img width="1290" alt="grafik" src="https://github.com/hashlips-lab/nft-erc721-collection/assets/104903064/727e3b3e-675e-426c-a806-6fa6b00512fd">

**Comment by liarco** [comment]
Hi @ClydeMaster, thank you for your feedback! 🙏
It looks like you have an error in your configuration, but I can't debug it from the screenshots you provided. 😥

At the moment we provide support to [Open Devs Crew](https://opendevs.io/) members only (except for bug reports and security issues) so if you need help I would suggest you to join our Discord private channel where you will be able to ask for support with higher priority. I hope to see you there! 💪 🚀

**Reply by ClydeMaster** [reply]
Hi Liarco!
Thanks a lot for your quick reply - really appreciate it!
Still do not know what happened, but sorted it out myself. Copied an older version and adapted it again. Dev Server works fine now. Will try yarn build later and see if it works correctly too. If not I will check the discord. You mean the Hashlips Discord, right? Am already a member of it :-).
Also, my project runs on Polygon. I would like to drop you one of my NFTs to say THANK YOU. Can I use the wallet address you provide on your GitHub profile?
All the best!

**Reply by liarco** [reply]
Awesome @ClydeMaster! My guess is you probably had a typo in your Tailwind config, but I didn't have it in your screenshots. I'm happy to hear you solved it 😃

The Open Devs Crew has a private channel on the HashLips Discord server (token gated with ODC NFTs) where members can get in touch with us directly (feature requests, support, questions, features requests, etc.). You can find all the info on [opendevs.io](https:/opendevs.io/).

About the address: yes, that one is correct, I removed it from your message because it may change in the future and I don't won't people to maybe find this discussion and use an old one, but the one you see on my profile page is always up to date. By the way, thank you for this, I appreciate it :)
